Toolset/palettes/behaviors/revaboutcardbehavior.livecodescript

```diff
@@ -0,0 +1,88 @@
+﻿script "revAboutCardBehavior"
+on preOpenStack
+   local tEdition, tPath
+   
+   lock screen
+   
+   switch revLicenseType()
+      case "commercial"
+         put "Indy Edition" into field "Edition"
+         break
+      case "community"
+         put "Community Edition" into field "Edition"
+         break
+      case "professional"
+         put "Business Edition" into field "Edition"
+         break
+      default
+         put "Commercial Edition" into field "Edition"
+   end switch
+   
+   put the version && "| Build" && the buildNumber into field "version"
+   set the blendLevel of field "version" to 40
+   
+   set the htmlText of field "Copyright" of me to "<p>&copy; 2000-" & word -1 of the long date && "LiveCode Ltd.</p>"
+   revFontApplySystemStyle the long id of field "Copyright" of me, empty, empty
+   
+   set the textColor of char 1 to -1 of field "displayData" of me to empty
+   set the textColor of char 1 to 5 of field "displayData" of me to aboutHiliteColor()
+   loadText "about"
+   
+   set the title of this stack to "About LiveCode" && line 4 of the revLicenseInfo
+   
+   if char 1 of the version < 7 then
+      put ideCoreFolderPath("skin", "about-background_legacy.png") into tPath
+      set the textColor of field "version" of me to "white"
+      set the textColor of field "edition" of me to "white"
+   else
+      put ideCoreFolderPath("skin", "about-background.png") into tPath
+      set the textColor of field "version" of me to empty
+      set the textColor of field "edition" of me to empty
+   end if
```

Comment:
These `ideCoreFolderPath`s need changing to use the revEnvironment API don't they?

Reply:
`ideCoreFolderPath` uses `revEnvironmentEditionProperty` so it's not necessary